src/riak_kv_pb_timeseries.erl

```diff
@@ -127,6 +130,53 @@ process(#tsputreq{table = Bucket, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Bucket, BucketProps), State}
     end;
 
+%% @ignore DELETE (as if)
+process(#tsdelreq{table = Table, key = CompoundKey_,
+                  vclock = VClock_, timeout = Timeout},
+        State) ->
+    Options =
+        lists:flatmap(
+          fun({_, undefined}) -> [];
+             ({_, default})   -> [];
+             ({K, V})         -> [{K,V}] end,
+          [{timeout, Timeout}]),
+    VClock = if VClock_ =:= undefined -> undefined;
```

Comment:
Another style nitpick, but we don't normally use `if` blocks unless it's a poor fit for a `case` statement. In this situation I think it would blend in better with the rest of the code and be slightly clearer to do something like this:
```
VClock = case VClock0 of
    undefined -> undefined;
    _ -> riak_object:decode_vclock(VClock0)
end,
```